### ISSUE TITLE
Site used to work but alas no more  - beeg.com

### ISSUE THREAD

**Issue by michaelgorman** [yt-dlp/yt-dlp#2592]
### Checklist

- [X] I'm reporting a new site support request
- [X] I've verified that I'm running yt-dlp version **2022.01.21**. ([update instructions](https://github.com/yt-dlp/yt-dlp#update))
- [X] I've checked that all provided URLs are alive and playable in a browser
- [X] I've checked that none of provided URLs [violate any copyrights](https://github.com/ytdl-org/youtube-dl#can-you-add-support-for-this-anime-video-site-or-site-which-shows-current-movies-for-free) or contain any [DRM](https://en.wikipedia.org/wiki/Digital_rights_management) to the best of my knowledge
- [X] I've searched the [bugtracker](https://github.com/yt-dlp/yt-dlp/issues?q=) for similar issues including closed ones. DO NOT post duplicates
- [X] I've read the [guidelines for opening an issue](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#opening-an-issue)
- [x] I've read about [sharing account credentials](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#are-you-willing-to-share-account-details-if-needed) and am willing to share it if required

### Region

US

### Example URLs

https://beeg.com/-0983946056129650

### Description

Used to be able to download using ...
yt-dlp --verbose --no-check-certificate --referer "https://beeg.com/-0983946056129650" "https://beeg.com/-0983946056129650"

It does require a free login which I can provide but it's easy enough just to create a new one

### Verbose log

```shell
[debug] Command-line config: ['--verbose', '--no-check-certificate', '--referer', 'https://beeg.com/-0983946056129650', 'https://beeg.com/-0983946056129650']
[debug] Encodings: locale cp1252, fs utf-8, out utf-8, err utf-8, pref cp1252
[debug] yt-dlp version 2022.01.21 [f20d607] (win_exe)
[debug] Python version 3.8.10 (CPython 64bit) - Windows-10-10.0.19042-SP0
[debug] exe versions: none
[debug] Optional libraries: Cryptodome, mutagen, sqlite, websockets
[debug] Proxy map: {}
[debug] [generic] Extracting URL: https://beeg.com/-0983946056129650
[generic] -0983946056129650: Requesting header
WARNING: [generic] Falling back on generic information extractor.
[generic] -0983946056129650: Downloading webpage
[generic] -0983946056129650: Extracting information
[debug] Looking for video embeds
ERROR: Unsupported URL: https://beeg.com/-0983946056129650
Traceback (most recent call last):
  File "yt_dlp\YoutubeDL.py", line 1367, in wrapper
  File "yt_dlp\YoutubeDL.py", line 1437, in __extract_info
  File "yt_dlp\extractor\common.py", line 612, in extract
  File "yt_dlp\extractor\generic.py", line 3966, in _real_extract
yt_dlp.utils.UnsupportedError: Unsupported URL: https://beeg.com/-0983946056129650
```

**Comment by nicolaasjan** [comment]
Worked here by using URL found via developer tools.
No login and referrer required.

**Comment by michaelgorman** [comment]
The only url I found was https://beeg.com/c7c879a4-2925-44d0-873c-6b056622af88 and I couldn't get it to work, disclaimer I'm not a web dev so I'm not sure if I'm looking in the right place

![image](https://user-images.githubusercontent.com/7871709/152162178-b29021d9-d21e-4a5f-80e3-d0382d582278.png)

**Comment by nicolaasjan** [comment]
Go to the "Network" tab, hit "Refresh" and then filter for "mp4".

![screenshot1](https://user-images.githubusercontent.com/14093220/152166196-f7d150f8-0a3c-4536-a3ab-58424b22b78c.png)

**Comment by michaelgorman** [comment]
Got it now thx for your help

**Comment by nicolaasjan** [comment]
Better leave this issue open, so that yt-dlp could update their extractor ([beeg.py](https://github.com/yt-dlp/yt-dlp/blob/master/yt_dlp/extractor/beeg.py)).
Then it wouldn't be necessary to use this workaround...

**Comment by michaelgorman** [comment]
reopening to have yt-dlp support it

**Comment by campones** [comment]
> Go to the "Network" tab, hit "Refresh" and then filter for "mp4".
> 
> ![screenshot1](https://user-images.githubusercontent.com/14093220/152166196-f7d150f8-0a3c-4536-a3ab-58424b22b78c.png)

actually for some reason this doesn't work using a linux computer (tried several ips)

**Comment by pukkandan** [comment]
This issue is fixed in #2616. Wait for next release
